### PR TITLE
Updated Network Policy how-to document

### PR DESCRIPTION
**Pull request by KashifSaadat**

### CODE REVIEW

File: how-to-docs/network-policies.md [hunk 1]
```diff
@@ -28,4 +28,48 @@ spec:
...
+  name: allow-dns-access
+  namespace: <your-namespace-here>
+spec:
+  podSelector:
+    matchLabels: {}
+  policyTypes:
+  - Egress
+  egress:
+  - to:
+    - namespaceSelector:
```
Comment (jaykeshur) [inline how-to-docs/network-policies.md]: you won't be able to resolve anything external with this. just svc.cluster.local addresses.

Comment (KashifSaadat) [reply how-to-docs/network-policies.md]: I think all DNS lookups would route via kube-dns in kube-system so it should still resolve external domains. You just won't be able to egress out to hit anything other than DNS (via kube-system)

Comment (KashifSaadat) [reply how-to-docs/network-policies.md]: Have a peek here at point 5 (I followed this guide in my testing): https://docs.projectcalico.org/master/getting-started/kubernetes/tutorials/advanced-policy . It shows successful resolution of Google

Comment (jaykeshur) [reply how-to-docs/network-policies.md]: when i tried it, i wasn't able to resolve google.com

Comment (jaykeshur) [reply how-to-docs/network-policies.md]: from kash: we we're missing: 
`kubectl label namespace kube-system name=kube-system`

Comment (jaykeshur) [reply how-to-docs/network-policies.md]: so guess that needs to be rolled out across our platform too

**Pull request merged**